.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: Java CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Run all tests
+    runs-on: ubuntu-18.04
+    timeout-minutes: 20
+    steps:
+    - name: Set up JDK 1.11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
+
+    - uses: actions/checkout@v1
+    
+    - name: Cache
+      uses: actions/cache@v1.1.0
+      with:
+        key: maven-cache
+        path: ~/.m2/repository
+    
+    - name: Install Maven Dependencies
+      run: mvn install -DskipTests=true -Dmaven.javadoc.skip=true --quiet --batch-mode --show-version --activate-profiles travis
```

Comment:
We should rename the profile in the parent pom to be `coveralls` or `open-source-ci` as `travis` is misleading now. Anyway, out of scope for now.